src/test/java/seedu/address/model/RoomBookTest.java

```diff
@@ -0,0 +1,94 @@
+package seedu.address.model;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.PriorityQueue;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.hotel.Room;
+
+
+
+class RoomBookTest {
+
+    @Test
+    void testEquals() {
+        PriorityQueue<Room> rooms = new PriorityQueue<>();
```

Comment:
Just curious, why did you choose to use PriorityQueue instead of something simpler like ArrayList? :P

Reply:
I see a lot of you have a lot of questions on PQ. I just thought that the comparison would be simpler as the priority is maintained by the PQ, and all we have to do is to poll and we dont have to compare a lot of variables so fewer bugs.